### PR TITLE
Quick switch / collapse orgs (optional patch using display:none)

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Use this PR as a followup to https://github.com/coordinape/coordinape/pull/2123 in the case that there is or avatar flickering in prod

### CODE REVIEW

File: src/features/nav/NavOrgs.tsx [hunk 1]
```diff
@@ -81,10 +87,115 @@ export const NavOrgs = ({
...
+              css={{
+                justifyContent: 'space-between',
+                width: '100%',
+                cursor: orgs.length > 1 ? 'pointer' : 'default',
+                '&:hover svg': { color: '$cta' },
+                '> div': { height: '$lg' },
+              }}
+            >
+              <NavLabel
+                key={'orgLabel'}
```
Comment (levity) [inline src/features/nav/NavOrgs.tsx:131]: same thing here: is the key needed? if so, use normal quotes pls

**Review by levity — CHANGES_REQUESTED** [review]

File: src/features/nav/NavCircles.tsx [hunk 2]
```diff
@@ -34,34 +33,68 @@ export const NavCircles = ({
...
+              org.myCircles.length > 1 &&
+              currentCircle && (
+                <IconButton
+                  css={{
+                    '&:hover': { color: '$cta' },
+                    rotate: viewCircleList ? '90deg' : 0,
+                    transition: '0.1s all ease-out',
+                  }}
+                >
+                  <Dots nostroke />
```
[the remark this answers is not in the document]
Comment (wingfeatherwave) [reply src/features/nav/NavCircles.tsx]: I'm OK with the chevron.  Thought the dots were fun...  But if it's potentially confusing, it's probably not worth it.  Changed to chevron.

File: src/features/nav/NavCircles.tsx [hunk 3]
```diff
@@ -34,34 +33,68 @@ export const NavCircles = ({
...
+            justifyContent: 'space-between',
+            width: '100%',
+            cursor:
+              org.myCircles.length > 1 && currentCircle ? 'pointer' : 'default',
+            '&:hover svg': { color: '$cta' },
+            '> div': { height: '$lg' },
+          }}
+        >
+          <NavLabel
+            key={'circlesLabel'}
```
[the remark this answers is not in the document]
Comment (wingfeatherwave) [reply src/features/nav/NavCircles.tsx]: key not needed.  removed

**Comment by crabsinger** [comment]
@wingfeatherwave I'm seeing some curious flickering behavior in the currently selected Org "Coordinape" in this case - captured in this video - when opening and closing the menu.

It looks like I see a "Co" without an image flicker in place during the transition.



https://user-images.githubusercontent.com/83605543/234063225-2debc3e1-d8c4-4afa-a807-ef8328aa5133.mov

**Comment by wingfeatherwave** [comment]
@crabsinger I've fixed the flicker.  It's not as elegant of code, but it seems to work

**Pull request closed without merging**